### PR TITLE
Rails 5 support and adapted Travis CI configuration

### PR DESCRIPTION
**Pull request by psylone**
Here are the changes from #8 (thanks to @kidbombay) and some adaptations for Travis CI.

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -37,8 +37,8 @@ def create_database
   ActiveRecord::Schema.define(:version => 1) do
     create_table :users do |t|
       t.string :name
-      t.string :colors, array: true, defualt: '{}'
-      t.timestamps
+      t.string :colors, array: true, default: []
```
Comment (Nerian) [inline spec/spec_helper.rb]: Should you update the Readme too?   https://github.com/psylone/acts-as-taggable-array-on/blob/1a7a9db771f44e0b62701152354035bf62964bb9/README.md#setup

Comment (psylone) [inline spec/spec_helper.rb]: Actually, both cases are the same:

```ruby
    create_table :users do |t|
      t.string :name
      t.string :tags,    array: true, default: '{}'
      t.string :friends, array: true, default: []

      t.timestamps
    end
```

Will look in Postgres:

```
                                     Table "public.users"
   Column   |            Type             |                     Modifiers
------------+-----------------------------+----------------------------------------------------
 id         | integer                     | not null default nextval('users_id_seq'::regclass)
 name       | character varying           |
 tags       | character varying[]         | default '{}'::character varying[]
 friends    | character varying[]         | default '{}'::character varying[]
 created_at | timestamp without time zone | not null
 updated_at | timestamp without time zone | not null
Indexes:
    "users_pkey" PRIMARY KEY, btree (id)
```

**Comment by psylone** [comment]
Any updates? Is this PR okay or we need to change something?

**Comment by tmiyamon** [comment]
Thank you for your contribution.

**Comment by kidbombay** [comment]
Woohoo! Nice work!

**Pull request merged**